src/datasets/search/search_clients_engines_sources_last_seen/intro.md

```diff
@@ -1,16 +1,16 @@
-`search_clients_last_seen` is designed for individual search analysis for clients over the past year.
+`search_clients_engines_sources_last_seen` is designed for individual search analysis for clients over the past year.
```

Comment:
Is it true that both the desktop and mobile search last_seen datasets are over a full year? If so, we have this misdocumented within bigquery-etl.

Reply:
Yes they both use 365 bits.  The mobile one is documented as 365 days